ui/src/overview.tsx

```diff
@@ -150,6 +174,9 @@ function ParticipantCriteria(props: { group: Group; criteria: Criteria }) {
       >
         Delete Criteria
       </MenuItem>,
+      <MenuItem key="2" onClick={showRenameCriteria}>
```

Comment:
not sure whether we should change the key = "2" to some string name like  "rename_criteria". but the reason why I write 2 is because the above is 1.